src/helpers/chartGradient.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Defs, LinearGradient, Stop } from 'react-native-svg';
+
+export default class ChartGradient extends React.Component {
+  constructor() {
```

Comment:
I don't think you need a constructor here or a class based component. You could easily use a function.